File: atcoder/abs/0.c
//usr/bin/env picoc $0 - $@;exit
#include <stdio.h>
int main(){
	int a,b,c;
	char s[999];
	scanf("%d%d%d%s",&a,&b,&c,s);
	printf("%d %s\n",a+b+c,s);
	return 0;
}

File: atcoder/abs/0.swift
//usr/bin/env swift $0 $@;exit
let a=Int(readLine()!)!
let arr=readLine()!.characters.split{$0==" "}.map{Int(String($0))!}
let b=arr[0]
let c=arr[1]
print(a+b+c,terminator:"")
print(" ",terminator:"")
print(readLine()!)
